### DISCUSSION TITLE
10x and Rhapsody integration

### DISCUSSION THREAD

**Discussion by JensFGG** [satijalab/seurat#5368] [Q&A]
Would it be possible to integrate 10x with rhapsody data? If yes, are there things I have to take into account? If not, why not :P?

**Comment by samuel-marsh** [comment]
If both are whole transcriptome or targeted using same targets then yes should be possible.  The same issues as most any other technology integration analysis will be present in terms of transcript capture method, efficiency, cell type capture, etc.  It will also like most other integrations be highly experiment specific so likely will just need to try some things out and see what it looks like.

Best,
Sam
